xla/service/latency_hiding_scheduler.cc

```diff
@@ -47,6 +47,7 @@ limitations under the License.
 #include "xla/hlo/ir/hlo_instructions.h"
 #include "xla/hlo/ir/hlo_opcode.h"
 #include "xla/hlo/ir/hlo_schedule.h"
+#include "xla/hlo/utils/hlo_query.h"
```

Comment:
Is this really needed? If I'm not mistaken, our internal IDE listed this unused at some point, but maybe I'm wrong.